packages/react/src/styles/base.css

```diff
@@ -0,0 +1,12 @@
+@import 'hds-design-tokens';
+@import 'hds-core/lib/utils/animations.css';
+@import 'hds-core/lib/utils/helpers.css';
```

Comment:
HDS doesn't work in hds-next project and there's console errors (404 Not found) about these specific packages and files. Is this hds-next settings problem or should this file be included somewhere in hds settings/build?

Reply:
Oh and it would be good to test this in SSR. How do we do that again?

Reply:
> HDS doesn't work in hds-next project and there's console errors (404 Not found) about these specific packages and files. Is this hds-next settings problem or should this file be included somewhere in hds settings/build?

I need to check this

> Oh and it would be good to test this in SSR. How do we do that again?

The site uses SSR support of HDS. I think we could test that with/without js

Reply:
> The site uses SSR support of HDS. I think we could test that with/without js
Oh right! Doc site works well and theming too. Without javascript it doesn't load anything though.

Reply:
I fixed this by addin postcss-import plugin into rollup config. It will inline imported css.